src/components/Card/Post.jsx

```diff
@@ -1,7 +1,7 @@
 import { React } from 'react';
 import image from '../../images/image-earth.jpeg';
 
-const Card = ({ title, text }) => (
+const Post = ({ title, text }) => (
```

Comment:
It could have the name Card, not necessary to rename it
At least, if you're going to rename it, why you did not rename the folder that contain this file?